### PR TITLE
switch to universal analytics

### PR DESCRIPTION
**Pull request by adamlofting**

### CODE REVIEW

File: app/http/views/site/account.html [hunk 1]
```diff
@@ -142,20 +142,13 @@ <h1 class="login-above">{{ gettext( 'Login above to access your account informat
...
-      _gaq.push(['_setDomainName', domain]);
-    {% endif %}
-    _gaq.push(['_trackPageview']);
-
-    (function() {
-      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-    })();
+    ga('create', '{{ GA_ACCOUNT }}', 'auto');
```
Comment (jbuck) [inline app/http/views/site/account.html]: ga_account needs to be lowercase

**Comment by jbuck** [comment]
@adamlofting r-

**Comment by adamlofting** [comment]
@jbuck nice catch!

R?

**Comment by jbuck** [comment]
r+

**Pull request merged**